### DISCUSSION TITLE
the sorted result of sidebar content is not equal.

### DISCUSSION THREAD

**Discussion by lmy668** [vuepress-theme-hope/vuepress-theme-hope#3943] [Q&A] [answered]
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/25795645/b11a0615-e861-4426-a8f9-bf2e18509658)
 see red square content

**Comment by ymp12345** [comment]
我也遇到了，个别文章侧边栏和目录生成的排序不一致

**Comment by Mister-Hope** [comment, marked as answer]
https://github.com/vuepress-theme-hope/vuepress-theme-hope/issues/3944
